src/components/shared/PrizeModal.tsx

```
@@ -119,57 +146,80 @@ export default function PrizeModal({ prize, isOpen, onClose }: PrizeModalProps)
+                border: `1px solid ${colors.primary}30`,
+              }}
+            >
               {/* Inner Modal Container */}
-              <div className={`${theme === 'light' ? 'bg-theme-background' : 'bg-black'} rounded-2xl sm:rounded-3xl overflow-hidden relative w-full h-full theme-transition`}>
+              <div
+                className={`${
+                  theme === "light" ? "bg-theme-background" : "bg-black"
+                } rounded-2xl sm:rounded-3xl overflow-hidden relative w-full h-full theme-transition`}
+                onKeyDown={handleModalKeyDown}
+                tabIndex={-1}
+              >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)